projects/ngx-formentry/src/form-entry/helpers/dataset-sample.ts

```diff
@@ -0,0 +1,313 @@
+export const womanDataset = [
```

Comment:
The reason I had comments in the example version of the table was to be able to track which table corresponded to what on the chart.

If we follow the WHO instructions for lookup, i.e., woman -> smoker -> age 70-74 -> SBP >= 180 (so the top-right row of the chart), the values should be [30, 31, 32, 33, 34]. For this row, this should always be a value in here.

Now, if we were to try to transcribe the full chart in the top right (female smoker age 70-74), we would have (in my original version):

```
[
  [14, 15, 15, 16, 17],
  [17, 18, 19, 19, 20],
  [21, 22, 22, 23, 24],
  [25, 26, 27, 28, 29],
  [30, 31, 32 33 34]
]
```

If we invert the SBP index, then the chart would look like this:

```
[
  [30, 31, 32 33 34],
  [25, 26, 27, 28, 29],
  [21, 22, 22, 23, 24],
  [17, 18, 19, 19, 20],
  [14, 15, 15, 16, 17],
]
```

However, if we follow the order this code is currently asking for things in than the chart doesn't really correspond to any sub-part of the written-out chart. You would need to have something like:

```
[
  [21, 22, 22, 23, 24],
  [30, 31, 32, 33, 34]
]
```

I think re-writing the chart is this way is ultimately going to be more error-prone because it's dividing up the chart in a somewhat odd way, making it difficult to determine whether what's here is correct or not.